core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -163,18 +165,19 @@ class SpendReportingService(userInfo: UserInfo, dataSource: SlickDataSource, big
              | GROUP BY currency, date
              |""".stripMargin
 
-          queryParams = List(
-            stringQueryParameter("billingAccountId", spendExportConf.billingAccountId.withoutPrefix()),
-            stringQueryParameter("startDate", dateTimeToISODateString(startDate)),
-            stringQueryParameter("endDate", dateTimeToISODateString(endDate)),
-            stringArrayQueryParameter("projects", workspaceProjects.map(_.value).toList)
-          )
-          jobRef <- bigQueryDAO.startParameterizedQuery(spendReportingServiceConfig.serviceProject, query, queryParams, "NAMED")
-          jobStatus <- bigQueryDAO.getQueryStatus(jobRef)
-          result: GetQueryResultsResponse <- bigQueryDAO.getQueryResult(jobStatus)
+          queryJobConfiguration = QueryJobConfiguration
+            .newBuilder(query)
+            .addNamedParameter("billingAccountId", stringQueryParameterValue(spendExportConf.billingAccountId.withoutPrefix()))
+            .addNamedParameter("startDate", stringQueryParameterValue(dateTimeToISODateString(startDate)))
+            .addNamedParameter("endDate", stringQueryParameterValue(dateTimeToISODateString(endDate)))
+            .addNamedParameter("projects", stringArrayQueryParameterValue(workspaceProjects.map(_.value).toList))
+            .build()
+
+          result <- bigQueryService.use(_.query(queryJobConfiguration)).unsafeToFuture()
```

Comment:
Why `unsafeToFuture`?

Reply:
`bigQueryService.use(_)` returns an `IO`, but this for comprehension requires that everything on the right side of the arrows be a `Future` (this is because the right side of the first line returns a `Future`). `unsafeToFuture` converts the `IO` to a `Future`. 

`unsafe` is the cats-effect way of indicating that whatever is in the `IO` is no longer contained within the "safe" confines of `IO`. In this case, that's because we're converting it to a `Future`. You'll also commonly see `unsafeRunSync` which is how you can force the evaluation of the `IO` and get its results. You can read more about it here: https://typelevel.org/cats-effect/docs/2.x/datatypes/io#introduction

And same offer as above, let me know if you want to talk through any cats-effect stuff at any point (or maybe cats-effect could be a good tech talk topic...)